src/registration/scripts/directives.js

```diff
@@ -13,6 +14,17 @@
                 link: function (scope, element, attrs) {
                     scope.data = {};
 
+                    // Bind on-success attribute to success callback
+                    // scope: { success: '&on-success' }
+                    if (angular.isDefined(attrs.onSuccess)) {
+                        var parentGet = $parse(attrs.onSuccess);
+                        if (parentGet !== angular.noop) {
```

Comment:
what if `parentGet` is not set...  i.e. is `undefined`? that won't match `angular.noop` and will probably generate an error?